pmg/admin/__init__.py

```diff
@@ -847,6 +847,62 @@ def on_model_change(self, form, model, is_created):
             tmp.from_upload(file_data)
             model.profile_pic_url = tmp.file_path
 
+    @expose("/attendance/")
+    def attendance(self):
+        """
+        """
+        mem_id = request.args.get("id")
+        url = "/admin/committeemeetingattendance/?member_id={0}".format(mem_id)
+        return redirect(url)
+
+
+class MemberCommitteeMeetingAttendanceView(MyModelView):
+    column_list = (
+        "meeting.committee.name",
+        "meeting.date",
+        "meeting.title",
+        "attendance",
+        "alternate_member",
+    )
+    column_labels = {
+        "meeting.date": "Meeting date",
+        "meeting.title": "Meeting title",
+        "meeting.committee.name": "Committee name",
+        "attendance": "Attendance code",
+    }
+    column_searchable_list = ("attendance",)
+    column_default_sort = ("meeting_id", True)
+    column_filters = ["meeting.date", "meeting.committee"]
+    column_formatters = {
+        "meeting.committee.name": lambda v, c, m, n: Markup(
+            "<a href='%s'>%s</a>"
+            % (
+                url_for("committee_meeting", event_id=m.meeting_id),
+                m.meeting.committee.name,
+            ),
+        ),
+        "meeting.date": lambda v, c, m, n: m.meeting.date.date().isoformat(),
+    }
+
+    def is_visible(self):
+        return False
```

Comment:
if it doesn't take more than 10 minutes, show under committee meetings here. It should def not be in main menu.
![Screenshot_2021-06-17_15-26-11](https://user-images.githubusercontent.com/235801/122405504-68720b00-cf80-11eb-9a0a-a55d1cbceffb.png)